sdk/src/wallet/account/operations/output_claiming.rs

```diff
@@ -318,10 +313,9 @@ where
                 };
                 // Recalculate every time, because new inputs can also add more native tokens, which would increase
                 // the required storage deposit
-                required_amount = required_amount_for_nfts
-                    + BasicOutputBuilder::new_with_amount(Output::AMOUNT_MIN)
-                        .with_native_tokens(option_native_token.into_iter().flatten())
-                        .rent_cost(rent_structure);
+                required_amount = BasicOutputBuilder::new_with_amount(required_amount_for_nfts)
+                    .with_native_tokens(option_native_token.into_iter().flatten())
+                    .min_storage_deposit_amount(rent_structure, token_supply)?;
```

Comment:
min_storage_deposit_amount() just overwrites `required_amount_for_nfts` and doesn't add to it?

Reply:
No, it adds to it using the logic from the sufficient fn